sympy/sets/fancysets.py

```diff
@@ -48,6 +48,10 @@ def _contains(self, other):
             return other.is_Rational
         return other.is_rational
 
+    def _eval_is_subset(self, other):
+        if other.is_subset(S.Integers):
+            return False
```

Comment:
I don't think that these lines are covered nor needed if it is handled in `is_subset_sets`

Reply:
These aren't needed in this PR any more. I am wondering if I should being back `_eval_is_subset` though. Having used multipledispatch now I can see that it is problematic to have many generic handlers like `Union, Set` because the dispatch mechanism can become ambiguous. Maybe for those cases it still makes sense to have a one-sided `_eval_is_subset`.